src/QuiqChatClient.js

```diff
@@ -74,6 +74,11 @@ class QuiqChatClient {
     return this;
   };
 
+  onWelcomeFormRegistration = (callback: () => void): QuiqChatClient => {
+    this.callbacks.onWelcomeFormRegistration = callback;
+    return this;
+  };
```

Comment:
Is this getting a little specific for QuiqChatCleint? Can we rename it to` onRegister`? It's a small issue, but in my mind we want to try and keep QuiqChatClient as general as possible...